dbt/adapters/sqlserver/__version__.py

```diff
@@ -1 +1 @@
-version = "1.3.0"
+version = "1.4.0"
```

Comment:
actions wywalają się z błędem:
Git tag: 1.4.0bee does not match the version of this app: 1.4.0
Error: Process completed with exit code 1. 

cyt. z teams: "suffix żeby nie generował buildów i potencjalnie nie mylił się w przyszłości z oficjalnym releasem" -> czyli nie przeszkadza nam błąd w releasie w GA? nie wiem czy rozumiem, co masz na mysli

Reply:
Jest tak jak piszesz, jeszcze patrzyłem i w sumie można byłoby tu wpisać tamtą wersję